src/build.sh

```diff
@@ -121,8 +121,10 @@ if [ "$SKIP_BASE" == 'no' ]; then
   build_base_image "ci-base-ldap:${TAG}"   base-ipa
   build_base_image "ci-base-ground:${TAG}" base-nfs
   build_base_image "ci-base-ground:${TAG}" base-kdc
+  build_base_image "ci-base-ground:${TAG}" base-keycloak
 fi
 
+set -e
```

Comment:
This shouldn't be needed, there's set -xe on the top of the file.

Reply:
removed